Dockerfile.nginx

```diff
@@ -0,0 +1,2 @@
+FROM nginx:1.11.13-alpine
```

Comment:
Yes the idea is to build a very small nginx server image with just the static files inside, now with some documentations, thanks for the feedback.